guides/common/modules/ref_best-practices-for-sync-plans.adoc

```diff
@@ -0,0 +1,23 @@
+[id="best-practices-for-sync-plans_{context}"]
+= Best practices for sync plans
+
+* Add sync plans to products and regularly synchronize content to avoid overly large changes in repositories.
+Synchronize content rather each night than only once a month.
```

Comment:
Should this be less specific
```suggestion
Synchronize content rather more often than less often.
```

Perhaps add an example of a day after that.

Reply:
Applied.